tinygrad/renderer/cstyle.py

```diff
@@ -220,6 +220,7 @@ def __init__(self): self.tensor_cores = MetalRenderer.tensor_cores if os.uname()
   float4 = "float4"
   uses_ptr_arithmetic = True
   code_for_workitem = {"g": lambda x: f"gid.{chr(120+x)}", "l": lambda x: f"lid.{chr(120+x)}"}
+  # uint3 used for gid/lid - TODO: this should probably be `ushort3 lid [[thread_position_in_threadgroup]]`
```

Comment:
yeah I remember RDNA3 reused VGPRs with bitshifting for lidx but not for gidx - maybe the compilers just handle it